package.json

```diff
@@ -69,8 +68,8 @@
     "svgstore": "2.0.2",
     "tap": "11.1.2",
     "through2": "2.0.3",
-    "vinyl-buffer": "1.0.0",
-    "vinyl-source-stream": "1.1.0",
+    "vinyl-buffer": "1.0.1",
+    "vinyl-source-stream": "2.0.0",
```

Comment:
Had to bump these packages since they didn't work properly after gulp upgrade. (See https://github.com/hughsk/vinyl-source-stream/issues/27)